config/routes.rb

```diff
@@ -1,6 +1,35 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  resources :likes
+  resources :comments
+  resources :followers
+  resources :photos
+  devise_for :users
+  devise_for :usernames
 
-  # Defines the root path route ("/")
   root "photos#index"
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
```

Comment:
You do not need any of these user routes. Devise takes care of these. Please delete these as well as your users controller

Reply:
done